pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -105,6 +109,22 @@ func (f *framework) RunReservePlugins(
 	return nil
 }
 
+// RunUnreservePlugins runs the set of configured unreserve plugins. If any of these
+// plugins returns an error, it does not continue running the remaining ones and
+// returns the error.
+func (f *framework) RunUnreservePlugins(
+	pc *PluginContext, pod *v1.Pod, nodeName string) *Status {
+	for _, pl := range f.unreservePlugins {
+		status := pl.Unreserve(pc, pod, nodeName)
+		if !status.IsSuccess() {
```

Comment:
We keep calling all Unreserve plugins regardless of the return value.

Reply:
Addressed